src/PopOver/PopOver.component.tsx

```diff
@@ -86,19 +112,21 @@ export class PopOver extends React.PureComponent<
             shadow,
             spacing = 8,
             maxWidth,
-            position = 'bottomStart',
+            position = 'bottom',
```

Comment:
Is this the default we want to change it to for popovers? iirc you changed this to make it consistent for debugging